### PR TITLE
Add support for Gams (.gms)

### PR DESCRIPTION
**Pull request by b-pos465**
http://www.gams.com/

### CODE REVIEW

File: docs/css-classes-reference.rst [hunk 1]
```diff
@@ -5,6 +5,13 @@ This is a full list of available classes corresponding to languages'
 syntactic structures. The parentheses after language name contain identifiers
 used as class names in ``<code>`` element.
 
+
+Gams ("gms", "gams")
+--------------------
+
+* ``keyword``:          keyword
+
```
Comment (isagalaev) [inline docs/css-classes-reference.rst]: The language also defines "string", "comment" and "number", please include those in the reference too.

File: src/languages/gams.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module.exports = function (hljs){
```
Comment (isagalaev) [inline src/languages/gams.js]: This won't let the build system to properly wrap this file. A language file should contain a bare JavaScript function and some meta data (http://highlightjs.readthedocs.org/en/latest/language-contribution.html#provide-meta-data).

File: src/languages/gams.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    var keywords = 'abort acronym acronyms alias all and assign binary card diag display else1 eps eq equation equations file ' +
+        'files for1 free ge gt if inf integer le loop lt maximizing minimizing model models na ne negative no not option   options or ord ' +
+        'parameter parameters positive prod putpage puttl repeat sameas scalar scalars semicont semiint set1 sets smax smin solve sos1 sos2 ' +
+        'sum system table then until using variable variables while1 xor yes';
+
+    keywords += keywords.toUpperCase();
```
Comment (isagalaev) [inline src/languages/gams.js]: It's easier to just set `case_insensitive: true` instead of duplicating keywords.

**Comment by Sannis** [comment]
@b-pos465 are you plan to update CSS reference and add some test snipper got Gams?

**Pull request merged**